sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/memory.scala

```diff
@@ -149,18 +149,12 @@ case class MemoryStream[A : Encoder](id: Int, sqlContext: SQLContext)
   }
 
   private def generateDebugString(
-      blocks: Iterable[Array[UnsafeRow]],
+      blocks: Seq[UnsafeRow],
```

Comment:
nit: it's probably more "rows" than "blocks" now

Reply:
right! i thought of changing but forgot. my bad.